Cargo.toml

```diff
@@ -66,6 +66,9 @@ pprof = { version = "0.9", features = ["criterion", "flamegraph"] } # MSRV 1.56
 bench = false
 
 [features]
+default = ["halo2-batch", "halo2-multicore"]
+halo2-batch = ["halo2_proofs/batch"]
+halo2-multicore = ["halo2_proofs/multicore"]
```

Comment:
```suggestion
multicore = ["halo2_proofs/multicore"]
```
This is then the same thing we do in `zcash_proofs`: https://github.com/zcash/librustzcash/blob/0cff3c67ed4f16368b5f34986dbe1834390b340b/zcash_proofs/Cargo.toml#L54